### PR TITLE
Cart and Wishlist management

### PR DESCRIPTION
**Pull request by Koushith**
## Description

- [x] Adds cart and wishlist management

## Screenshot
(https://www.loom.com/share/d1435d54e7a2481cb779872f2ae108fd)

### CODE REVIEW

File: src/screens/my-cart/containers/cart-items/cart-item.component.js [hunk 1]
```diff
@@ -23,16 +29,38 @@ export const CartItem = (props) => {
             <p className={cartItem.discount_percentage}>({discountCalc(items.price, items.originalPrice)}% OFF)</p>
           </div>
           <div className={cartItem.quantity_container}>
-            <button>-</button>
-            <input type='number' value='2' />
```
Comment (sashaboi) [inline src/screens/my-cart/containers/cart-items/cart-item.component.js:27]: in this input , a styling of text-align center might make it look better ?

**Review by sashaboi — APPROVED** [review]
It would usually be nice to have a preview link to play with the site yourself, hope that comes in the future.
The styling is AMAZING , the colors, the on hover properties, etc looks spectacular. Home page and image selection I love what you've done ! 

Only thought of one suggestion , not super mandatory but might improve UI :)

**Review by praveenreddy1798 — APPROVED** [review]

File: src/screens/my-cart/containers/cart-items/cart-item.component.js (hunk 1 above)
Comment (Koushith) [reply src/screens/my-cart/containers/cart-items/cart-item.component.js:27]: yeah, I will work o it.

**Comment by Koushith** [comment]
> thought of one suggestion , not super mandatory but might impro

Thanks for the feedback. not sure for some reason build failed. will try to add the deployment link.

**Pull request merged**